dot/services_test.go

```diff
@@ -381,3 +384,25 @@ func Test_createPprofService(t *testing.T) {
 
 	require.NotNil(t, service)
 }
+
+func Test_createDigestHandler(t *testing.T) {
+	testDatadirPath := t.TempDir()
+	config := state.Config{
+		Path:     testDatadirPath,
+		LogLevel: log.Info,
```

Comment:
nit perhaps change that to a higher level like

```suggestion
		LogLevel: log.Warn,
```

So our tests are less log spammed by default? :thinking:

Reply:
Make sense